### PR TITLE
Mention OpenAPI docs in Galaxy API Documentation

### PR DESCRIPTION
**Pull request by bernt-matthias**
how to access api docs

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

File: doc/source/api_doc.rst [hunk 1]
```diff
@@ -18,7 +18,15 @@ The Galaxy API addresses these and other situations by exposing Galaxy
...
-Various language specific bindings for interfacing with the Galaxy API have been developed by the Galaxy community including `Python <https://bioblend.readthedocs.io/>`__, `PHP <https://github.com/galaxyproject/blend4php>`__, `Java <https://github.com/jmchilton/blend4j>`__, and `JavaScript <https://github.com/galaxyproject/galaxy/tree/dev/client/src/mvc>`__.
+The up-to-date Galaxy API documentation can be accessed by appending ``api/docs``
+to the base URL of a Galaxy server, e.g. https://usegalaxy.org/api/docs
+
+Various language specific bindings for interfacing with the Galaxy API have been
+developed by the Galaxy community including `Python
+<https://bioblend.readthedocs.io/>`__, `PHP
+<https://github.com/galaxyproject/blend4php>`__, `Java
+<https://github.com/jmchilton/blend4j>`__, and `JavaScript
+<https://github.com/galaxyproject/galaxy/tree/dev/client/src/mvc>`__.
```
Comment (mvdbeek) [inline doc/source/api_doc.rst]:
```suggestion
<https://github.com/jmchilton/blend4j>`__, and `TypeScript
<https://github.com/galaxyproject/galaxy/tree/dev/client/src/schema>`__.
```

File: doc/source/api_doc.rst [hunk 2]
```diff
@@ -18,7 +18,15 @@ The Galaxy API addresses these and other situations by exposing Galaxy
...
 
-Various language specific bindings for interfacing with the Galaxy API have been developed by the Galaxy community including `Python <https://bioblend.readthedocs.io/>`__, `PHP <https://github.com/galaxyproject/blend4php>`__, `Java <https://github.com/jmchilton/blend4j>`__, and `JavaScript <https://github.com/galaxyproject/galaxy/tree/dev/client/src/mvc>`__.
+The up-to-date Galaxy API documentation can be accessed by appending ``api/docs``
+to the base URL of a Galaxy server, e.g. https://usegalaxy.org/api/docs
+
+Various language specific bindings for interfacing with the Galaxy API have been
+developed by the Galaxy community including `Python
+<https://bioblend.readthedocs.io/>`__, `PHP
+<https://github.com/galaxyproject/blend4php>`__, `Java
+<https://github.com/jmchilton/blend4j>`__, and `JavaScript
```
Comment (martenson) [inline doc/source/api_doc.rst]:
```suggestion
<https://github.com/galaxyproject/blend4j>`__, and `JavaScript
```

File: doc/source/api_doc.rst [hunk 3]
```diff
@@ -18,7 +18,15 @@ The Galaxy API addresses these and other situations by exposing Galaxy
 internals through an additional interface, known as an Application Programming
 Interface, or API.
 
-Various language specific bindings for interfacing with the Galaxy API have been developed by the Galaxy community including `Python <https://bioblend.readthedocs.io/>`__, `PHP <https://github.com/galaxyproject/blend4php>`__, `Java <https://github.com/jmchilton/blend4j>`__, and `JavaScript <https://github.com/galaxyproject/galaxy/tree/dev/client/src/mvc>`__.
+The up-to-date Galaxy API documentation can be accessed by appending ``api/docs``
```
Comment (nuwang) [inline doc/source/api_doc.rst]: Should `/redoc` be included as well?

Comment (martenson) [inline doc/source/api_doc.rst]:
```suggestion
The up-to-date Galaxy API documentation can be accessed by appending ``/api/docs`` or ``/redoc``
```

Comment (mvdbeek) [reply doc/source/api_doc.rst]: if we do that it should be hosted at `/api/redoc`

Comment (bernt-matthias) [reply doc/source/api_doc.rst]: If someone tells me how to change this (if easy :) ) then I can change it .. or just push to this branch. 

For me the most important point is that one (the one mentioned in the release notes) is documented.

Comment (mvdbeek) [reply doc/source/api_doc.rst]: https://fastapi.tiangolo.com/tutorial/metadata/#docs-urls

**Comment by martenson** [comment]
https://github.com/bernt-matthias/galaxy/pull/6

**Review by mvdbeek — APPROVED** [review]

**Comment by davelopez** [comment]
Thank you all!

**Pull request merged**